policyengine_us/variables/gov/states/mo/tax/income/deductions/mo_pension_and_ss_or_ssd_deduction_section_b.py

```diff
@@ -30,16 +30,13 @@ def formula(person, period, parameters):
                 - p.mo_private_pension_deduction_allowance[filing_status]
             ),
         )
-        ind_pvt_pension_amt = person("taxable_private_pension_income", period)
-        ind_pvt_pension_val = min_(
-            ind_pvt_pension_amt,
-            p.mo_max_private_pension,
-        )
-        unit_pvt_pension_val = tax_unit.sum(ind_pvt_pension_val)  # line8
-        unit_deduction = max_(0, unit_pvt_pension_val - excess_agi)  # line9
-        ind_share_of_unit_deduction = where(
-            ind_pvt_pension_val > 0,
-            ind_pvt_pension_val / unit_pvt_pension_val,
-            0,
-        )
-        return ind_share_of_unit_deduction * unit_deduction
+        ind_pvt_pen_amt = person("taxable_private_pension_income", period)
+        ind_pvt_pen_val = min_(ind_pvt_pen_amt, p.mo_max_private_pension)
+        unit_pvt_pen_val = tax_unit.sum(ind_pvt_pen_val)  # line8
+        unit_deduction = max_(0, unit_pvt_pen_val - excess_agi)  # line9
+
```

Comment:
Since this syntax might be unfamiliar to other devs could we add a comment like this in other locations too?
```suggestion
    # Compute the individual's share of the tax unit's taxable private pension income.
    # Use a mask rather than where to avoid a divide-by-zero warning.
```